daml-lf/spec/transaction.rst

```diff
@@ -155,6 +155,8 @@ This table lists every version of this specification in ascending order
 +--------------------+-----------------+
 |                  5 |      2019-03-12 |
 +--------------------+-----------------+
+|                  6 |      2019-04-15 |
```

Comment:
This version must be added to the `VersionTimeline` and `TransactionVersions`.

Reply:
Additionally there'll need to be a case to pick `minExerciseReturnValue` in `TransactionVersions.assignVersion`; you can use the last argument to `latestWhenAllPresent` therein as a template for how to do that.

Reply:
Done